src/main/kotlin/skytils/skytilsmod/features/impl/mining/MiningFeatures.kt

```diff
@@ -194,29 +194,26 @@ class MiningFeatures {
              * $asdf:Khazad-dûm@-292,63,281 ❌
              * $SBECHWP:Khazad-dûm@asdf,asdf,asdf ❌
              */
-            val cleanedUnformatted = unformatted.removeExceptPattern(SBE_DSM_PATTERN)
-            val matcher = SBE_DSM_PATTERN.matcher(cleanedUnformatted)
-            if (matcher.matches()) {
-                val stringLocation = matcher.group("stringLocation")
-                val x = matcher.group("x")
-                val y = matcher.group("y")
-                val z = matcher.group("z")
-                val loc = CrystalHollowsMap.Locations.values().find { it.cleanName == stringLocation }
-                if (loc != null) {
-                    if (!loc.loc.exists()) {
-                        /**
-                         * Sends the waypoints message except it suggests which one should be used based on
-                         * the name contained in the message and converts it to the internally used names for the waypoints.
-                         */
-                        UMessage("§3Skytils > §eFound coordinates in a chat message, click a button to set a waypoint.\n")
-                            .append(UTextComponent("§f${loc.displayName} ")
-                                .setClick(MCClickEventAction.RUN_COMMAND, "/skytilshollowwaypoint set ${loc.id} $x $y $z")
-                                .setHoverText("§eSet waypoint for ${loc.displayName}"))
-                            .append(UTextComponent("§e[Custom]")
-                                .setClick(MCClickEventAction.SUGGEST_COMMAND, "/skytilshollowwaypoint set name_here $x $y $z")
-                                .setHoverText("§eSet custom waypoint")
-                            ).chat()
-                    }
+            val cleaned = SBE_DSM_PATTERN.find(unformatted)
+            if (cleaned != null) {
+                println("cleaned = ${cleaned.value}")
```

Comment:
Remove this log line or replace it with a dev message `printDevMessage()`

Reply:
Done!